plugins/lxc/lxc_ram

```diff
@@ -62,13 +70,18 @@ if [ "$1" = "config" ]; then
  echo 'graph_vlabel byte'
  echo 'graph_category memory'
 
+ if [ "$areastack" = "true" ]; then
+  for guest_name in $guest_names; do
+    guest="$(clean_fieldname $guest_name)"
 
- for guest_name in $guest_names;
-  do
-   guest="$(clean_fieldname $guest_name)"
+    echo 'mem_usage_'$guest'.label '$guest_name': Mem usage'
```

Comment:
The quoting of this line (and the ones following) is a bit hard to read (and I think it is misleading).
I assume that you just want to do the following?
```
echo "mem_usage_$guest.label $guest_name: Mem usage"
```
Single quotes: no variable expansion inside.
Double quotes: variable expansion happens inside.